Strange error with security-jdbc:
```
2023-02-23T13:11:25.0803953Z 2023-02-23 13:11:24,784 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-0) HTTP Request to %s failed, error id: %s [Error Occurred After Shutdown]: java.lang.RuntimeException: org.wildfly.security.auth.server.RealmUnavailableException: ELY01050: Could not execute query "SELECT u.password, roles FROM test_user u WHERE u.username=?"
2023-02-23T13:11:25.0805106Z 	at io.quarkus.elytron.security.runtime.ElytronPasswordIdentityProvider$1.get(ElytronPasswordIdentityProvider.java:62)
2023-02-23T13:11:25.0806009Z 	at io.quarkus.elytron.security.runtime.ElytronPasswordIdentityProvider$1.get(ElytronPasswordIdentityProvider.java:43)
2023-02-23T13:11:25.0872488Z 	at io.quarkus.security.runtime.QuarkusIdentityProviderManagerImpl$1$1$1$1.run(QuarkusIdentityProviderManagerImpl.java:58)
2023-02-23T13:11:25.0904737Z 	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:576)
2023-02-23T13:11:25.0905512Z 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
.......
2023-02-23T13:11:25.1071836Z Caused by: org.h2.jdbc.JdbcSQLDataException: Invalid value "3" for parameter "columnIndex" [90008-214]
2023-02-23T13:11:25.1072273Z 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:646)
2023-02-23T13:11:25.1072694Z 	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477
.......

```

The interesting message there is that `[Error Occurred After Shutdown]`, not sure what that means, there might be some race condition, the server is being shutdown while security jdbc is still doing something.

I also think it is not related, can you rebase though just to give it another try, as a newer wildfly-security version is available on main ?

--- a/extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/standalone/ResteasyStandaloneRecorder.java
+++ b/extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/standalone/ResteasyStandaloneRecorder.java
@@ -17,6 +17,7 @@ import io.quarkus.runtime.RuntimeValue;
 import io.quarkus.runtime.ShutdownContext;
 import io.quarkus.runtime.annotations.Recorder;
 import io.quarkus.security.AuthenticationCompletionException;
+import io.quarkus.security.AuthenticationException;
 import io.quarkus.security.AuthenticationFailedException;
 import io.quarkus.security.AuthenticationRedirectException;
 import io.quarkus.security.ForbiddenException;
@@ -126,9 +127,7 @@ public class ResteasyStandaloneRecorder {
                         }
                     }
 
-                    if (request.failure() instanceof AuthenticationFailedException
-                            || request.failure() instanceof AuthenticationCompletionException
-                            || request.failure() instanceof AuthenticationRedirectException
+                    if (request.failure() instanceof AuthenticationException
                             || request.failure() instanceof ForbiddenException) {
                         super.handle(request);
                     } else {

--- a/extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/ResteasyReactiveRecorder.java
+++ b/extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/ResteasyReactiveRecorder.java
@@ -53,6 +53,7 @@ import io.quarkus.runtime.RuntimeValue;
 import io.quarkus.runtime.ShutdownContext;
 import io.quarkus.runtime.annotations.Recorder;
 import io.quarkus.security.AuthenticationCompletionException;
+import io.quarkus.security.AuthenticationException;
 import io.quarkus.security.AuthenticationFailedException;
 import io.quarkus.security.AuthenticationRedirectException;
 import io.quarkus.security.ForbiddenException;
@@ -255,9 +256,7 @@ public class ResteasyReactiveRecorder extends ResteasyReactiveCommonRecorder imp
                     }
                 }
 
-                if (event.failure() instanceof AuthenticationFailedException
-                        || event.failure() instanceof AuthenticationCompletionException
-                        || event.failure() instanceof AuthenticationRedirectException
+                if (event.failure() instanceof AuthenticationException
                         || event.failure() instanceof ForbiddenException) {
                     restInitialHandler.beginProcessing(event, event.failure());
                 } else {

--- a/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/QuarkusErrorHandler.java
+++ b/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/QuarkusErrorHandler.java
@@ -18,9 +18,7 @@ import org.jboss.logging.Logger;
 import io.netty.handler.codec.http.HttpHeaderNames;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.quarkus.runtime.TemplateHtmlBuilder;
-import io.quarkus.security.AuthenticationCompletionException;
-import io.quarkus.security.AuthenticationFailedException;
-import io.quarkus.security.AuthenticationRedirectException;
+import io.quarkus.security.AuthenticationException;
 import io.quarkus.security.ForbiddenException;
 import io.quarkus.security.UnauthorizedException;
 import io.quarkus.vertx.http.runtime.security.HttpAuthenticator;
@@ -82,9 +80,7 @@ public class QuarkusErrorHandler implements Handler<RoutingContext> {
                 return;
             }
 
-            if (event.failure() instanceof AuthenticationFailedException
-                    || event.failure() instanceof AuthenticationCompletionException
-                    || event.failure() instanceof AuthenticationRedirectException) {
+            if (event.failure() instanceof AuthenticationException) {
 
                 if (event.response().getStatusCode() == HttpResponseStatus.OK.code()) {
                     //set 401 if status wasn't set upstream

